package.json

```diff
@@ -21,18 +21,18 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^4.14.0",
-    "@theforeman/eslint-plugin-foreman": "^4.14.0",
-    "@theforeman/stories": "^4.14.0",
-    "@theforeman/test": "^4.14.0",
-    "@theforeman/vendor-dev": "^4.14.0",
+    "@theforeman/builder": "^8.7.0",
```

Comment:
Let's split this into another PR? current version is 8.10.0

Reply:
https://github.com/theforeman/foreman_remote_execution/pull/646